sass/components/_channel-switcher.scss

```
@@ -94,12 +94,17 @@
             position: absolute;
             width: calc(100% - 48px);
             margin: -28px 0 0 24px;
-            background: rgba(var(--center-channel-bg-rgb), 1);
+            background: #fff;
             box-shadow: $elevation-4;
             border-radius: 4px;
-            border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+            border: 1px solid #ddd;
```

Same here (not sure if the opacity is right here)
```suggestion
            border: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.16);
```